cell_classification/segmentation_data_prep_test.py

```diff
@@ -1,24 +1,40 @@
+from calendar import c
+from genericpath import exists
 import os
 import pytest
 import tempfile
 import numpy as np
+import pandas as pd
+import json
 from tifffile import imwrite
 from segmentation_data_prep import SegmentationTFRecords
 
 
 def prep_object():
     data_prep = SegmentationTFRecords(
-        data_folders="path_to_folders",
-        cell_table_path="path_to_cell_table",
+        data_folders=["list", "of", "data", "folders"],
+        cell_table_path="list_to_cell_table_path",
         conversion_matrix_path="path_to_conversion_matrix",
-        imaging_platform="ImagingPlatform",
-        dataset="Dataset",
+        imaging_platform="imaging_platform",
+        dataset="dataset",
         tile_size=[256, 256],
-        tf_record_path="path_to_tf_record",
+        tf_record_path="tf_record_path",
+        normalization_dict_path="path_to_normalization_dict",
     )
     return data_prep
 
 
+def prepare_conversion_matrix():
+    col_names = ["cluster_labels", "CD11c", "CD14", "CD56", "CD57"]
```

Comment:
There's a couple places in the code where you have this logic: checking the column names to remove one that is really a label for the rows. Is there a reason to have this extra column name here?

```
cols = ['CD11c', 'CD4', 'CD8']
rows = ['Tcell', 'Bcell', 'tumor']
data = np.random.randint(0, 2, size=(len(rows), len(cols)))
df = pd.DataFrame(data, columns=cols, index=rows)
```

This will mean we can always assume all column names are valid

Reply:
It's just how I imagined the `conversion_matrix` might look like when set up in excel. I can get rid of the extra column name and store the labels in the row names if this is desired.